lib/crawler_web/controllers/page_controller.ex

```diff
@@ -2,7 +2,5 @@
 defmodule CrawlerWeb.PageController do
```

Comment:
Can you please remove `credo:disable-for-this-file` from here and from all other files. It is better to fix the code than disable rules. I missed this line in the previous PR.

Reply:
Fixed in [ad04701](https://github.com/vnntsu/elixir-ic/pull/32/commits/ad047011f81fa1690a0b30b06d42c2428b04616d)